Sources/Datadog/RUM/RUMMonitor/RUMCommand.swift

```diff
@@ -23,10 +24,28 @@ internal struct RUMStartViewCommand: RUMCommand {
     /// The object (typically `UIViewController`) identifying the RUM View.
     let identity: AnyObject
 
+    /// The path of this View, rendered in RUM Explorer.
+    let path: String
+
     /// Used to indicate if this command starts the very first View in the app.
     /// * default `false` means _it's not yet known_,
     /// * it can be set to `true` by the `RUMApplicationScope` which tracks this state.
     var isInitialView = false
+
+    init(time: Date, identity: AnyObject, path: String?, attributes: [AttributeKey: AttributeValue]) {
+        self.time = time
+        self.attributes = attributes
+        self.identity = identity
+        self.path = path ?? RUMStartViewCommand.viewPath(from: identity)
+    }
+
+    private static func viewPath(from id: AnyObject) -> String {
+        guard let viewController = id as? UIViewController else {
+            return ""
+        }
+
+        return "\(type(of: viewController))"
```

Comment:
Today we provide API for `UIKit`, but it will be also `SwiftUI` tomorrow. I consider `UIViewController` in public API + `Any` in internal as a good compromise to avoid huge refactoring when we start working with `SwiftUI`. We just cast the thing at the very top and at the very bottom - everywhere internally we work with `Any`.

Reply:
yes, maybe i wasn't clear but that's what i'm saying: all the internals can be `Any`, why do we need to to cast `id` to `UIViewController` here?

Reply:
I see 👌. Yes, I removed the `UIViewController` casting here along with `import UIKit` - resulting with less `UIKit` coupling in RUM internals 👍.